wdn/templates_4.1/scripts/tabs.js

```diff
@@ -64,6 +66,70 @@ define(['jquery', 'wdn', 'require'], function($, WDN, require) {
 			nsel = trig.closest('li').siblings('.' + selected);
 			trig.trigger('tabchanged', [curr, nsel, tabs]);
 		},
+		
+		setupA11y = function ($tabsWithSwitch) {
+			$tabsWithSwitch.attr('role', 'tablist');
+
+			var $tabs = $($tabsWithSwitch.find('a'));
+
+			$tabs.each(function(index, element) {
```

Comment:
Unless index is used, it's easier just to use no arguments in the callback and rely on the callback context `this` being the element.